### DISCUSSION TITLE
UWP with modern .NET

### DISCUSSION THREAD

**Discussion by jp-weber** [unoplatform/uno#18379] [General]
Are there any plans yet on how to deal with the UWP and the support of the latest dotnet? Is support for the Uno SDK being considered?
https://devblogs.microsoft.com/ifdef-windows/preview-uwp-support-for-dotnet-9-native-aot/

**Comment by jeromelaban** [comment]
Thanks for the question. 

Not at this time.

This change by Microsoft is mostly meant for devs to move to WinUI more smoothly, and the current UWP support on our end is mostly there for that purpose, but adding this new target will be a very large headache to maintain, for a very small crowd. Additionally, UWP in Uno is very likely to be dropped in the next Uno Platform major version.

**Reply by jp-weber** [reply]
Thanks for the quick reply.

That's understandable. I only asked because I currently still have an UWP project next to a Uno platform project and this didn't work with the Uno.SDK and central nuget management. In future, however, the UWP project is to be completely replaced by the WinUI3 app 👍
